model/task_lifecycle.go

```diff
@@ -1080,6 +1081,8 @@ func UpdateBlockedDependencies(ctx context.Context, dependencies []task.Task) er
 }
 
 // UpdateUnblockedDependencies recursively marks all unattainable dependencies as attainable.
+// The updateAllDependencies indicates whether all tasks that depend on the given tasks will
+// be updated (i.e., whether the task's Dependency.Status will be ignored).
```

Comment:
Stray copy-paste? It looks like this parameter would have no effect for unblocking.